phenol-annotations/src/main/java/org/monarchinitiative/phenol/annotations/assoc/Gene2DiseaseAssociationParser.java

```diff
@@ -87,10 +87,10 @@
    */
   private void parseMim2geneAndGeneInfo(File homoSapiensGeneInfoFile, File mim2geneMedgenFile) {
     if (!homoSapiensGeneInfoFile.exists()) {
-      throw new PhenolRuntimeException("Cannot find Homo_sapiens.gene_info.gz file");
+      throw new PhenolRuntimeException("Cannot find Homo_sapiens.gene_info.gz file at \"" + homoSapiensGeneInfoFile + "\"");
     }
     if (!mim2geneMedgenFile.exists()) {
-      throw new PhenolRuntimeException("Cannot find mim2gene_medgen file");
+      throw new PhenolRuntimeException("Cannot find mim2gene_medgen file at \"" + homoSapiensGeneInfoFile + "\"");
```

Comment:
Shouldn't this read `mim2geneMedgenFile`?

Reply:
agree